test/e2e/upg_e2e.go

```diff
@@ -386,14 +386,14 @@ var _ = ginkgo.Describe("Multiple PFCP Sessions", func() {
 					seids = append(seids, seid)
 				}
 
-				ginkgo.By("deleting 100 sessions")
+				ginkgo.By("deleting 10000 sessions")
 				for _, seid := range seids {
 					deleteSession(f, seid, false)
 				}
 			}
 
-			ginkgo.By("Waiting 10 seconds for the queues to be emptied")
-			time.Sleep(10 * time.Second)
+			ginkgo.By("Waiting 40 seconds for the queues to be emptied")
+			time.Sleep(40 * time.Second)
```

Comment:
40 seconds timeout will surely work even on weak performing envs?

Reply:
When UPG receives a PFCP request, it caches the response for 30s. If the request is repeated (same seq), the cached response is sent again. After 30s, the entries from the cache are gone, and that is important for checking the leaks.